src/metrics/mod.rs

```diff
@@ -67,6 +78,64 @@ metrics! {
     namespace: "docsrs",
 }
 
+#[derive(Debug, Default)]
+pub(crate) struct RecentlyAccessedReleases {
+    krates: DashMap<i32, Instant>,
```

Comment:
This could be `crates` too.
```suggestion
    crates: DashMap<i32, Instant>,
```